sdks/python/run_pylint.sh

```diff
@@ -23,7 +23,7 @@
 #
 # The exit-code of the script indicates success or a failure.
 
-set -e
+set -o errexit
```

Comment:
using `set -o errexit` is preferable, since it is more explicit and thus more readable